webapp/packages/plugin-connections-administration/src/Administration/Connections/ConnectionsTable/Connection.tsx

```diff
@@ -40,9 +40,10 @@ const styles = css`
 interface Props {
   connectionKey: IConnectionInfoParams;
   connection: DatabaseConnection;
+  projectName?: string | null;
```

Comment:
do we really need undefined and null at the same time?
better to keep undefined and remove the null